### ISSUE TITLE
a11y: Color Contrast - Login

### ISSUE THREAD

**Issue by rak-phillip** [rancher/dashboard#13463]
Insufficient color contrast for standard text

## Instances

"Foreground color: #3D98D3
Background color: #FFFFFF
Contrast ratio:  3.16:1
Main Content
- [ ] Username (In typing state)
- [ ] Password (In typing state)
- [ ] Show (in default state)
- [ ] Hide (in default state, available after activating Show) 
- [ ] English (in default state)

Foreground color: #FFFFFF
Background color: #3D98D3
Contrast ratio:  3.16:1
Main Content
- [ ] Log in with Local User 

Foreground color: #F64747
Background color: #FEE4E4
Contrast ratio:  2.94:1
Main Content
- [ ] Invalid username or password. Please try again.  (Available after the submission of the blank form) 

Foreground color: #5D995D
Background color: #FFFFFF
Contrast ratio:  3.39:1
Main Content
- [ ] You have been logged out. (Available when user logged out)

Foreground color: #DCDEE7
Background color: #FFFFFF
Contrast ratio:  1.34:1
Main content
- [ ] Remember Username (unselected checkbox)

## Actual result

- The standard text with background/foreground color combinations having an insufficient contrast ratio making it difficult for people with low vision and color blindness to read standard text elements.

## Expected result

The contrast ratio between foreground text and background for the standard text should be sufficient.

## Recommendation

On all pages, ensure that each foreground/background color combination presents a contrast ratio of at least 4.5:1 for standard text.

## Screenshots

![Image](https://github.com/user-attachments/assets/4df818b0-53cf-486b-a315-417e70a429cf)
![Image](https://github.com/user-attachments/assets/a082d844-3261-434a-a4c7-8bc0d108d3a5)
![Image](https://github.com/user-attachments/assets/d89d7837-3aee-4b78-95f1-724e947d394e)
![Image](https://github.com/user-attachments/assets/63f7dadb-291c-477c-a781-f31246db343b)
![Image](https://github.com/user-attachments/assets/d08ee224-502f-4ddc-bf72-eb3fca251274)

**Comment by rak-phillip** [comment]
Fails the VPAT:

> The mentioned checkbox border have insufficient contrast with their background, making it difficult for people with low vision and color blindness to distinguish them.

Issue 50
